webapp/components/User/index.spec.js

```diff
@@ -77,7 +77,7 @@ describe('User', () => {
         it('renders anonymous user', () => {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T17:55:46Z" title="Thursday, May 2nd 2019, 7:55:46 pm +02:00">May 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -74,11 +75,13 @@ describe('User', () => {
           propsData.user.disabled = true
         })
 
-        it('renders anonymous user', () => {
+        // TODO recheck what we want to display if a user is disabled
+        // it seems not reasonable to diplay Anonymous
```


Together with @datenbrei we decided that we will not publish that a user was disabled. Instead we show sth. that says "User not available". @ulfgebhardt do you mean that "Anonymous" is not the best description for that? Feel free to use "User not available".

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-06T11:12:50Z" title="Monday, May 6th 2019, 1:12:50 pm +02:00">May 6, 2019</time>_

---


Disabled Users are not displayed at all for normal Users.
Users with Admin/Mod privilege will see the User normally (with red Border)

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-06T11:13:38Z" title="Monday, May 6th 2019, 1:13:38 pm +02:00">May 6, 2019</time>_

---


![image](https://user-images.githubusercontent.com/1238238/57221806-c17cc000-7000-11e9-94f1-d1f59e3b8688.png)

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-06T11:13:38Z" title="Monday, May 6th 2019, 1:13:38 pm +02:00">May 6, 2019</time>_

---


![image](https://user-images.githubusercontent.com/1238238/57221806-c17cc000-7000-11e9-94f1-d1f59e3b8688.png)

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-06T11:14:34Z" title="Monday, May 6th 2019, 1:14:34 pm +02:00">May 6, 2019</time>_

---


On Profiles of other users there is a Problem tho: 
![image](https://user-images.githubusercontent.com/1238238/57221843-e2451580-7000-11e9-8297-a61fbbec9a0b.png)